src/client/editor/caches/GLTFCache.js

```diff
@@ -72,7 +39,7 @@ class GLTFCache extends Cache {
             if (prop instanceof THREE.Texture) {
               if (prop.image.src) {
                 const absoluteTextureURL = new URL(prop.image.src, window.location).href;
-                textureCache._cache.set(absoluteTextureURL, Promise.resolve(prop));
+                this.textureCache._cache.set(absoluteTextureURL, Promise.resolve(prop));
```

Comment:
i thought our convention was `_` are private members?

Reply:
Yeah, I probably should make cache public then. Thanks